mantidproject/mantid
# Fix linux pre commit

*Marking High Priority, as we currently have PRs going through formatting CI effectively untested*

**Description of work.**
Fixes the git commit hook which was only checking the first file due to `$1` instead of `$@` to pass all args

**To test:**
- Checkout PR
- `pre-commit run clang-format --all-files` previously took <1s, it should now take at least >10s
- Try changing a random file and re-running with the all-files flag. It should now be picked up

~If you are happy with this could you ping me to get the corresponding [PR](https://github.com/mantidproject/pre-commit-hooks/pull/8/commits/4a3e150e80aaa0f2810af279279862817547e1a2) merged, then update the hook SHA / location~
Edit: merged

There is no associated issue. - Noticed when my PRs kept randomly breaking :frowning_face: 





---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
